content/influxdb3/core/reference/config-options.md

```diff
@@ -777,6 +779,31 @@ Defines the address on which InfluxDB serves HTTP API requests.
 
 ---
 
+#### admin-token-recovery-http-bind
+
+Enables an admin token recovery HTTP server on a separate port. This server allows regenerating lost admin tokens without existing authentication. The server automatically shuts down after a successful token regeneration.
+
+> [!Warning]
+> This option creates an unauthenticated endpoint that can regenerate admin tokens. Only use this when you have lost access to your admin token and ensure the server is only accessible from trusted networks.
+
+**Default:** `127.0.0.1:8182` (when enabled)
+
+| influxdb3 serve option | Environment variable |
+| :--------------------- | :------------------- |
+| `--admin-token-recovery-http-bind` | `INFLUXDB3_ADMIN_TOKEN_RECOVERY_HTTP_BIND` |
+
+##### Example usage
+
+```bash
+# Start server with recovery endpoint
+influxdb3 serve --admin-token-recovery-http-bind
```

Comment:
Ok, this option appears to work differently than other options. If you include just the option, does it default to `127.0.0.1:8182`, or do you have to specify a value? AFAIK, options either require a value or are a boolean option.

Reply:
> Ok, this option appears to work differently than other options. If you include just the option, does it default to `127.0.0.1:8182`, or do you have to specify a value? AFAIK, options either require a value or are a boolean option.

This option can be used with or without specifying an address:
- Without value: Uses default address `127.0.0.1:8182`
- With value: Uses the specified address